### DISCUSSION TITLE
Is tabby actually open source?

### DISCUSSION THREAD

**Discussion by traverseda** [TabbyML/tabby#2243] [Q&A] [answered]
It looks like there's no way currently to run tabby without proprietary code from the enterprise-edition, I think this makes tabby as a whole no longer an open source project?

**Comment by wsxiaoys** [comment]
it's still possible - you can pass `--no-webserver` flag to disable all enterprise related features.

See https://github.com/TabbyML/tabby/releases/tag/v0.11.0 for details

**Reply by traverseda** [reply]
It's nice that there's a simple flag for this. I do feel like there's some confusing language on your pricing page, that frankly comes across as a bit skeezy.

 * Strongly mentions open source on tabblyML web site, no mention of pricing
 * Pricing information and limitations only become visible once you launch the actual container
 * Community edition relies on closed-source components, something I've never seen before with any other open source project.

I think it's important to be clear about these kinds of expectations. I don't think you're intentionally misrepresenting this project, but the current language really does seem like that. Obviously you can do whatever you want, but I have some suggestions on how to can better manage people's expectations. 

 * Have the pricing page on your website, not just on your app.
 * Don't call your proprietary edition "Community Edition", that is some confusing language. I'd suggest calling it something like "Professional edition"
 * Add a column to your comparison called either "Community Edition" or "Open Source Edition", which features only FOSS code
 * Add two new rows to your comparison, a pricing row and an "Open Source" row

**Reply by wsxiaoys** [reply, marked as answer]
Thank you—that's certainly a valuable suggestion. We haven't had the chance to revamp the website documentation, as the whole enterprise offering is relatively new and was enabled by default only recently. However, adding a comparison page to clarify what's offered in Tabby OSS (`--no-webserver`) compared to Tabby is definitely something we want to do. This will be followed up with a document enhancement soon.
